### PR TITLE
Add Support for Wacom MTE-450, like #482 asked

### PR DESCRIPTION
**Pull request by C0D3-M4513R**

### CODE REVIEW

**Comment by X9VoiD** [comment]
For verification purposes, post screenshot of tablet debugger showing highest possible `Position`, and another for `Pressure`.

Also upload a diagnostics file, again.

**Comment by C0D3-M4513R** [comment]
I will, however the support, that this config provides is right now fluctuating massively. I need to find a better config. I will post results, when I get there. Have been trying to get it to run since then.

**Comment by C0D3-M4513R** [comment]
It is right now, as if OpenTabletDriver is not actually the active driver for the tablet.

**Comment by X9VoiD** [comment]
Uninstall the official wacom driver, close OTD, unplug the tablet, replace your config with this one

```js
{
  "Name": "Wacom MTE-450",
  "DigitizerIdentifiers": [
    {
      "Width": 147.6,
      "Height": 92.3,
      "MaxX": 14760.0,
      "MaxY": 9225.0,
      "MaxPressure": 511,
      "ActiveReportID": {
        "Start": null,
        "StartInclusive": false,
        "End": null,
        "EndInclusive": false
      },
      "VendorID": 1386,
      "ProductID": 101,
      "InputReportLength": 8,
      "OutputReportLength": null,
      "ReportParser": null,
      "FeatureInitReport": "AgI=",
      "OutputInitReport": null,
      "DeviceStrings": {},
      "InitializationStrings": []
    },
    {
      "Width": 147.6,
      "Height": 92.3,
      "MaxX": 14760.0,
      "MaxY": 9225.0,
      "MaxPressure": 511,
      "ActiveReportID": {
        "Start": null,
        "StartInclusive": false,
        "End": null,
        "EndInclusive": false
      },
      "VendorID": 1386,
      "ProductID": 101,
      "InputReportLength": 9,
      "OutputReportLength": null,
      "ReportParser": "OpenTabletDriver.Vendors.SkipByteTabletReportParser",
      "FeatureInitReport": "AgI=",
      "OutputInitReport": null,
      "DeviceStrings": {},
      "InitializationStrings": []
    }
  ],
  "AuxilaryDeviceIdentifiers": [],
  "Attributes": {}
}
```
Restart OTD, then do the verification steps.

**Comment by C0D3-M4513R** [comment]
It works without the `"FeatureInitReport": "AgI=",` too. I'm leaving that in there, since I don't exactly know, what that does

**Comment by C0D3-M4513R** [comment]
grafik

**Comment by C0D3-M4513R** [comment]
[sdgfgsdf.txt](https://github.com/InfinityGhost/OpenTabletDriver/files/5609453/sdgfgsdf.txt)

**Comment by InfinityGhost** [comment]
> It works without the `"FeatureInitReport": "AgI=",` too. I'm leaving that in there, since I don't exactly know, what that does

In very few cases, such as if the Wacom driver is installed it can help to prepare the device to read data.

Btw `AgI=` is Base64 for `0x02 0x02` lol

**Comment by InfinityGhost** [comment]
Before this gets merged, I want to notify you that we plan on switching licenses from GPLv3 to LGPLv3. If you don't agree to the change, we won't merge this pull request.

**Comment by C0D3-M4513R** [comment]
I agree

**Comment by InfinityGhost** [comment]
@X9VoiD @C0D3-M4513R is this correctly tested and ready for merge?

**Comment by C0D3-M4513R** [comment]
@InfinityGhost 
Have been using this config for 2 days without issues. Pen position, pen touch strength and pen buttons are working as well.
The tablet buttons are working as well. 

The only unsupported feature of the tablet would be, that the zoom circle is actually being detected as 2 buttons, which actually don't do anything.

**Comment by InfinityGhost** [comment]
I'm mostly concerned about the dimensions being correctly calculated, LPI and width/height should be matching the MaxX and MaxY. If we can confirm this we're g2g

File: OpenTabletDriver/Configurations/Wacom/MTE-450.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "Name": "Wacom MTE-450",
+  "DigitizerIdentifiers": [
+    {
+      "Width": 147.6,
+      "Height": 92.3,
```
Comment (X9VoiD) [inline OpenTabletDriver/Configurations/Wacom/MTE-450.json:6]:
```js
"Height": 92.25
```

File: OpenTabletDriver/Configurations/Wacom/MTE-450.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      "OutputReportLength": null,
+      "ReportParser": null,
+      "FeatureInitReport": "AgI=",
+      "OutputInitReport": null,
+      "DeviceStrings": {},
+      "InitializationStrings": []
+    },
+    {
+      "Width": 147.6,
+      "Height": 92.3,
```
Comment (X9VoiD) [inline OpenTabletDriver/Configurations/Wacom/MTE-450.json:28]:
```js
"Height": 92.25
```

**Review by X9VoiD — CHANGES_REQUESTED** [review]

**Comment by C0D3-M4513R** [comment]
> I'm mostly concerned about the dimensions being correctly calculated, LPI and width/height should be matching the MaxX and MaxY. If we can confirm this we're g2g

14760 MaxX/ 2540 lpi * 25,4(conversion from i to mm) =147,6 mm width
9225 MaxX: 2540 lpi * 25,4(conversion from i to mm) =92,25 mm height

**Review by X9VoiD — APPROVED** [review]

**Pull request merged**